portal/migrations/versions/3c871e710277_.py

```diff
@@ -152,49 +157,68 @@ def upgrade():
                 # no change needed in this situation
                 continue
 
-            # report if dates don't match spreadsheet in IRONN-210
-            cd_str = '{dt.day}-{dt:%b}-{dt:%y}'.format(dt=consent_date)
-            wd_str = '{dt.day}-{dt:%b}-{dt:%y}'.format(dt=withdrawal_date)
-            try:
-                match = verified_user_consent_dates[study_id][patient_id]
-                if (cd_str, wd_str) != match:
-                    print(f"user_id {patient_id} \t {cd_str} \t {wd_str}")
-                    print(" vs expected:")
-                    print(f"\t\t {match[0]} \t {match[1]}")
-            except KeyError:
-                # user found to not see timeline change
-                pass
-
-            # fake an adherence cache run to avoid unnecessary and more
-            # important, to prevent from locking out a subsequent update
-            # needed after recognizing a real change below
-            adherence_cache_moderation = CacheModeration(key=ADHERENCE_DATA_KEY.format(
-                patient_id=patient_id,
-                research_study_id=study_id))
-            adherence_cache_moderation.run_now()
-
-            b4_state = capture_patient_state(patient_id)
-            update_users_QBT(
-                patient_id,
-                research_study_id=study_id,
-                invalidate_existing=True)
-            _, _, _, any_changes = present_before_after_state(
-                patient_id, study_id, b4_state)
-            if not any_changes:
-                continue
+            if slow_report_details:
```

Comment:
I don't see this ever being set to true...

Reply:
> I don't see this ever being set to true...

no, that was just a convenient way to fence out that code, in case we wanted to review the noisy output.  that's exactly how i generated the files attached to IRONN-210 - but given that we're now hitting every withdrawn patient, it's an overwhelming amount of data (and more than doubles the time).

Reply:
👍